### PR TITLE
Generate unique key for event in multiple rooms

### PR DESCRIPTION
**Pull request by mhib**
@adamzima @s1mplex @jtkpiotr @lkudyba @quirk0o

### CODE REVIEW

File: client/app/components/calendar/event/EventGroup.jsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ export default class EventGroup extends React.Component {
       <Event event={event}
              containerHeight={containerHeight}
              unitEventLengthInSeconds={unitEventLengthInSeconds}
-             key={event.id}
+             key={`${event.id}|${event.conference_room.id}`}
```
Comment (adamzima) [inline client/app/components/calendar/event/EventGroup.jsx]: Why `|`?

Comment (mhib) [inline client/app/components/calendar/event/EventGroup.jsx]: I thought it might be a good idea to separate these ids.

Comment (adamzima) [inline client/app/components/calendar/event/EventGroup.jsx]: So why not `_`?

**Comment by adamzima** [comment]
Nice commit message.

**Pull request merged**